src/components/Carousel/Carousel.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <section class="carousel">
+    <div class="carousel__container">
+      <button
+        id="left-btn"
+        class="carousel-control carousel-control--left"
+        @click="handlePrevious"
+      >
+        <img
+          src="../../assets/icons/left.svg"
+          alt="left arrow"
+        >
+      </button>
+      <div
+        id="carousel-images"
+        class="carousel-images"
+      >
+        <CarouselItem
+          v-for="(photo, index) in photos"
```

Comment:
Consider having more appropriate name as now the code implies that you have an array of photos, although, these are shop items. 🤔  Maybe just name them `items` or `carouselItems`?